src/main/java/org/jboss/windup/operator/controller/WindupController.java

```diff
@@ -30,10 +30,19 @@ public class WindupController implements Watcher<WindupResource> {
 	@Inject
 	KubernetesClient k8sClient;
 
+	@ConfigProperty(name = "RELATED_IMAGE_MTR_WEB")
+	String webConsoleImage;
+
+	@ConfigProperty(name = "RELATED_IMAGE_MTR_WEB_EXECUTOR")
+	String executorImage;
```

Comment:
Why are we using the downstream brand `MTR` name for these variables? I suggest using `WINDUP` in any variable or just not use any brand name at all.